menu_konsolowe.py

```diff
@@ -1,3 +1,66 @@
+from CoWork_klasy import *
+import json
+
+
+def show_admin_desks_view():
+    print("Admin Desks View:")
+    for keys, values in desks_instances.items():
+        print(keys, values)
+
+
+def add_desk():
+    name = input("Podaj nazwę biurka: ")
+    desk_type = input("Podaj typ biurka: ")
+    price = float(input("Podaj cenę biurka PLN/h: "))
+    status = input("Podaj status biurka: ")
+
+    new_desk = Desks(name, desk_type, price, status)
+    desks_instances[name] = new_desk
+    print("Dodano biurko!")
+
+
+def delete_desk():
+    print("Aktualne Biurka:")
+    for idx, (name, desk) in enumerate(desks_instances.items()):
+        print(f"{idx}. {name} - {desk.desk_type} - {desk.price} PLN/h - {desk.status}")
+
+    if not desks_instances:
+        print("Brak dostępnych biurek do usunięcia.")
+        return
+
+    try:
+        del_desk = int(
+            input("Podaj indeks biurka do usunięcia (lub -1, aby anulować): ")
+        )
+
+        if del_desk == -1:
+            print("Anulowano usunięcie.")
+            return
+
+        if 0 <= del_desk < len(desks_instances):
+            desk_name = list(desks_instances.keys())[del_desk]
```

Comment:
bez sensu to rzutowanie na lsite usuwaj po kluczu del dict[key_name] albo pop, rzutowanie na lsite kompletnie nie potrzebne i tylko komplikuje :)